test_sets/API/vehicle_data.txt

```diff
@@ -16,6 +16,7 @@
 ./test_scripts/API/VehicleData/OnVehicleData/006_OnVD_mandatory_parameters.lua
 ./test_scripts/API/VehicleData/OnVehicleData/007_OnVD_min_max_boundary_values.lua
 ./test_scripts/API/VehicleData/OnVehicleData/008_OnVD_enum_and_bool_values.lua
+./test_scripts/API/VehicleData/OnVehicleData/009_PoliciesManager_must_allow_all_requested_VD_params_in_case_parameters_field_is_omitted.lua
```

Comment:
Let's rename this script to conform other ones for VD. E.g. `009_OnVD_allowed_for_omitted_parameters_field.lua`

Reply:
@dboltovskyi done, thanks